pyogrio/geopandas.py

```diff
@@ -103,16 +105,15 @@ def read_dataframe(
     sql_dialect : str, optional (default: None)
         The sql dialect the sql statement is written in. Possible values:
 
-          - **None**: if the datasource natively supports sql, the specific
-            sql syntax for this datasource should be used (eg. SQLite,
-            PostgreSQL, Oracle,...). If the datasource doesn't natively
-            support sql, the 'OGRSQL_' dialect is the
+          - **None**: if the datasource natively supports SQL, its specific SQL dialect
+            will be used by default (eg. SQLite and Geopackage: `SQLITE`_, PostgreSQL).
+            If the datasource doesn't natively support SQL, the `OGRSQL`_ dialect is the
             default.
-          - 'OGRSQL_': can be used on any datasource. Performance can suffer
+          - '`OGRSQL`_': can be used on any datasource. Performance can suffer
             when used on datasources with native support for sql.
-          - 'SQLITE_': can be used on any datasource. All spatialite_
+          - '`SQLITE`_': can be used on any datasource. All spatialite_
```

Comment:
```suggestion
          - '`SQLITE`_': can be used on any data source. All spatialite_
```

(and remaining instances below)